app/views/docs/databases-relationships.phtml

```diff
@@ -0,0 +1,100 @@
+
+<h2><a href="/docs/databases-relations#types" id="types">Types of Relationships</a></h2>
+<p>Appwrite provides four different relationship types to enforce different associative rules between documents.</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>One to one</code></td>
+            <td>A document can only be related with one and only one document. For example, each user has only one profile.</td>
+        </tr>
+        <tr>
+            <td><code>One to many</code></td>
+            <td>A document can be related to many other documents. For example, a user can create many posts.</td>
+        </tr>
+        <tr>
+            <td><code>Many to one</code></td>
+            <td>Many documents can be related to a single document. For example, many posts can share the same author.</td>
+        </tr>
+        <tr>
+            <td><code>Many to many</code></td>
+            <td>A document can be related to many other documents. For example, a user can have many friends and many users can share the same friend.</td>
+        </tr>
+    </tbody>
+</table>
+
+<p>Appwrite relationships can be directional or two-way. This helps [TODO: @jake why did we make these directional?].</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>Directional</code></td>
+            <td>[@jake can you elaborate on the practical impact of this? Maybe example on use case]</td>
+        </tr>
+        <tr>
+            <td><code>Two-way</code></td>
+            <td>[@jake can you elaborate on the practical impact of this? Maybe example on use case]</td>
+        </tr>
+    </tbody>
+</table>
+<p>
+    Appwrite also allows you to define the behavior of a relationship when a document is updated or deleted.
+</p>
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 250px">Type</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>Restrict</code></td>
+            <td>[@jake can you elaborate on the practical impact of this? Maybe example on use case]</td>
+        </tr>
+        <tr>
+            <td><code>Cascade</code></td>
+            <td>[@jake can you elaborate on the practical impact of this? Maybe example on use case]</td>
```

Comment:
@abnegate

Reply:
If you try to delete a document that has related at least one related document, the related documents will also be deleted. E.g. if I have a one-to-many relation between movies and reviews, and I try to delete a movie that has reviews, all that movies reviews will also be deleted. If I try to delete a movie with no reviews it will work with no extra deletions. If I try to delete a review related to a movie it will work with no extra deletions.